code/Assbin/AssbinExporter.cpp

```diff
@@ -413,7 +413,8 @@ class AssbinExport
 
         Write<unsigned int>(&chunk,tex->mWidth);
         Write<unsigned int>(&chunk,tex->mHeight);
-        chunk.Write( tex->achFormatHint, sizeof(char), 4 );
+        // Write the texture format, but don't include the null terminator.
+        chunk.Write( tex->achFormatHint, sizeof(char), HINTMAXTEXTURELEN - 1 );
```

Comment:
I think this would be a format change. Not sure how stable the binary format is supposed to be but this probably needs at least a major version bump.

Reply:
@turol It looks like the changes have been [merged](https://github.com/assimp/assimp/pull/2948). I am afraid I do not know if backward compatibility is important to Assimp devs/maintainers but this could be discussed further.